enterprise/frontend/src/metabase-enterprise/hosting/index.js

```diff
@@ -1,11 +1,11 @@
 import _ from "underscore";
 import { updateIn } from "icepick";
 import { t } from "ttag";
-import MetabaseSettings from "metabase/lib/settings";
 import { PLUGIN_ADMIN_SETTINGS_UPDATES } from "metabase/plugins";
-import { SettingsCloudStoreLink } from "../../components/SettingsCloudStoreLink";
+import { hasPremiumFeature } from "metabase-enterprise/settings";
+import { SettingsCloudStoreLink } from "./components/SettingsCloudStoreLink";
 
-if (MetabaseSettings.isHosted()) {
+if (hasPremiumFeature("hosting")) {
```

Comment:
Semantically this condition is different.
It seems that before it was "is hosted" and now it is "can be hosted".
Or does `hasPremiumFeature` mean `usesPremiumFeature`?

Reply:
@kamilmielnik it's the same. I just make this code look like other EE plugins. Technically the condition would give the same results because

```
(defsetting is-hosted?
  "Is the Metabase instance running in the cloud?"
  :type       :boolean
  :visibility :public
  :setter     :none
  :getter     (fn [] (boolean ((token-features) "hosting")))
  :doc        false)
```